packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -76,8 +76,10 @@ describe('ComboBox', () => {
   isConformant({
     Component: ComboBox,
     displayName: 'ComboBox',
-    // Disabled due to being required to mount by safeCreate. A test called "renders" was added here as a replacement.
-    disabledTests: ['component-renders'],
+    // Problem: Currently doesn’t work with mount.
+    // Solution: Further investigate why ComboBox only works with create or fix customMount type
```

Comment:
what's the error you are getting?
(note: this is only a test issue. ref is working correctly)

Reply:
When ComboBox uses enzyme mount it returns `Cannot read property 'tag' of null`. I'm not certain of the reason behind this but it seems to work correctly when using create.

Reply:
oh i know why. see this change I have [here](https://github.com/microsoft/fluentui/pull/15534/files#diff-4f25026635039c045750d43d45e8e7ca20b1141746eeafdaab93a93fde2e2cdd). `mount` relays on `createPortal` but combobox tests are mocking the `createPortal`. if you reset `createPortal` to original implementation before running the conformance tests, that should help resolve